db/schema.rb

```diff
@@ -344,6 +340,29 @@
     t.index ["user_id"], name: "index_samples_on_user_id"
   end
 
+  create_table "samples_visualizations", id: false, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
```

Comment:
sorry I know the staging db is messed up but could you edit the schema.rb to just have these two create_table blocks